src/components/stylesPageTemplate/index.js

```diff
@@ -0,0 +1,112 @@
+import { makeStyles } from '@material-ui/core/styles';
```

Comment:
Move "stylesPageTemplate" to "pageTemplate" folder.
And rename "stylesPageTemplate" folder to "styles".
If you use styled components name will be better "stylesComponents"

Reply:
Ok.